divinediscipline/send-it
# #161733902 Set up project structure for server and Apis

**What does this PR do?**
Set up project folders, and server

**Description of task to be completed**
-implement basic project set up and folders
-set up server
-set up default api routes

**How should this be manually tested?**
Clone the project, run "npm install" to install dependencies, run "node server/helpers/www" to execute the entry point file, the server should be running on port 5000.

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
[Delivers #161733902]
